tordnscrypt/src/main/res/values/strings.xml

```diff
@@ -627,85 +627,85 @@
+    <string name="update_over_five_times">You checked for new versions of InviZible Pro more than 5 times in a day. Please try again later.</string>
 
-    <string name="verifier_error">Looks like this unofficial version of InviZible. Please be care of it!</string>
-    <string name="only_for_pro">Only for PRO version</string>
+    <string name="verifier_error">This is unofficial version of InviZible. It was not built by the developer.</string>
+    <string name="only_for_pro">Only for the Pro version</string>
     <string name="donate">Donate</string>
-    <string name="donate_project">InviZible Pro Project needs your assistance. Please visit donate page or enter received premium code.</string>
+    <string name="donate_project">The InviZible Project needs your help. Please visit the donate page or enter your received premium code.</string>
     <string name="donate_button">Visit</string>
     <string name="enter_code_button">Enter Code</string>
     <string name="enter_code">Enter Code</string>
-    <string name="thanks_for_donate">InviZible Pro Project and its author express appreciation
-        for the assistance. Would you like to download and upgrade to PRO? The update will continue in the background.</string>
-    <string name="thanks_for_donate_gp">InviZible Pro Project and its author express appreciation for the assistance!</string>
+    <string name="thanks_for_donate">The InviZible Pro Project and its author appreciates
+        all help. Would you like to download and upgrade to Pro? The update will continue in the background.</string>
+    <string name="thanks_for_donate_gp">The InviZible Project and its author thanks you for your assistance.</string>
     <string name="pref_dnscrypt_relays_summ">Remote lists of available relays.</string>
 
-    <string name="vpn_mode_active">VPN mode is active</string>
+    <string name="vpn_mode_active">VPN mode is on</string>
     <string name="vpn_mode_off">VPN mode is off</string>
-    <string name="vpn_mode_error">VPN mode error!</string>
-    <string name="vpn_tor_tether_summ">Please configure the use of proxy server in the settings of the connected device. Default values: IP:10.1.10.1 Port:8118</string>
+    <string name="vpn_mode_error">VPN-mode error!</string>
+    <string name="vpn_tor_tether_summ">Please make use of a proxy server in the settings of the connected device. Default IP 10.1.10.1 at port 8118.</string>
 
-    <string name="main_fragment_button_start" translatable="false">START</string>
-    <string name="main_fragment_button_stop" translatable="false">STOP</string>
+    <string name="main_fragment_button_start" translatable="false">Start</string>
+    <string name="main_fragment_button_stop" translatable="false">Stop</string>
     <string name="main_fragment_hide_ip">Hide IP with TOR</string>
     <string name="main_fragment_protect_dns">Protect DNS with DNSCRYPT</string>
-    <string name="main_fragment_access_i2p">Access to I2P sites with Purple I2P</string>
```

Comment:
All "sites" (and WebSites) is now "websites". Not sure if "sites" is something I2P uses.